src/ert/scheduler/lsf_driver.py

```diff
@@ -150,10 +152,14 @@ async def submit(
             script_path = Path(script_handle.name)
         assert script_path is not None
         script_path.chmod(script_path.stat().st_mode | stat.S_IEXEC)
+        arg_resource_request = (
+            ["-R", f"'{self._resource_request}'"] if self._resource_request else []
```

Comment:
Is the single quoting here needed? Should it be responsibility of the user? Can the quotes even be injected via the config?

Reply:
It seems to do so in the old driver. Is LSF_SUBMIT_REMOTE_SHELL the submit method we will be using, or the one we are dropping with the scheduler?
```
char *quoted_resource_request = NULL;
    if (req) {
        if (driver->submit_method == LSF_SUBMIT_REMOTE_SHELL) {
            quoted_resource_request = saprintf("\"%s\"", req);
            free(req);
        } else
            quoted_resource_request = req;
    }
    return quoted_resource_request;
```

Reply:
We are dropping the remote shell feature yes.

If there can be spaces in resource requests, it should be tested in integration tests.